examples/plot_udct.py

```diff
@@ -0,0 +1,48 @@
+"""
+Uniform Discrete Curvelet Transform
+===================================
+This example shows how to use the :py:class:`pylops.signalprocessing.UDCT` operator to perform the
+Uniform Discrete Curvelet Transform on a (multi-dimensional) input array.
+"""
+
+import numpy as np
+from ucurv import udct, zoneplate, ucurvfwd, ucurvinv, ucurv2d_show
+import matplotlib.pyplot as plt
+import pylops
+plt.close("all")
+
+sz = [512, 512]
+cfg = [[3, 3], [6, 6]]
```

Comment:
Try to add some additional text explaining what you are doing, right now it is a bit hard to follow the example